api/src/main/java/com/datastrato/gravitino/SupportsMetalakes.java

```diff
@@ -2,10 +2,8 @@
  * Copyright 2024 Datastrato Pvt Ltd.
  * This software is licensed under the Apache License version 2.
  */
-package com.datastrato.gravitino.rel;
```

Comment:
This rename is also happened in https://github.com/datastrato/gravitino/pull/3419